app/assets/stylesheets/kudos/components/_tab-bar.scss

```diff
@@ -0,0 +1,23 @@
+@import '../utils/variables';
+
+select.drop_down_menu {
+  border:0;
+  -webkit-appearance: none;
+  font-size: 1.5em;
+  align-content: center;
+  font-weight: 600;
+  width: 100%;
+  height: 35px;
+  font-family: $font-gotham;
+  padding: 0 2px;
+  border: none;
+  background: transparent
+  url("data:image/svg+xml;utf8,
+  <svg
+  xmlns='http://www.w3.org/2000/svg'
+  width='20'
+  height='10'
+  fill='black'><polygon
+  points='0,2.5 15,2.5 7.5,10'/>
+  </svg>") no-repeat right;
+}
```

Comment:
If you hover over the little icon here you'll see "No newline at end of file".  Here's some background on why it's considered good style to have a newline at the end: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Do you know if your editor is configurable to automatically put newlines at the end of files?